proto/api/v1/resource_collection_service.proto

```diff
@@ -25,7 +25,7 @@ message GetCollectionRequest {
 
 message CollectionDeploymentMatchOptions {
     bool with_matches = 1;
-    Pagination matches_pagination = 2;
+    RawQuery filter_query = 3;
```

Comment:
You do not need to update the tag number. The tag `2` is still not used by end-users.

Reply:
Ah good catch, I forgot to do that